pkg/stores/sqlproxy/proxy_store.go

```diff
@@ -26,17 +25,20 @@ import (
 	apitypes "k8s.io/apimachinery/pkg/types"
 	"k8s.io/apimachinery/pkg/util/sets"
 	"k8s.io/apimachinery/pkg/watch"
+	"k8s.io/apiserver/pkg/endpoints/request"
 	"k8s.io/client-go/dynamic"
 	"k8s.io/client-go/kubernetes"
 	"k8s.io/client-go/rest"
 	"k8s.io/client-go/tools/cache"
 
```

Comment:
Can we remove this dangling EOL and reorder imports?

```suggestion
	"k8s.io/client-go/tools/cache"
```